requirements-dev.txt

```diff
@@ -3,6 +3,6 @@ pytest~=6.2.4
 pytest-vcr~=1.0.2
 sphinx~=4.1.2
 rst2pdf~=0.98
-black~=21.7b0
+black~=22.3.0
```

Comment:
previous version was failing due to an issue with import incompatibility when running tests. Updated to resolve